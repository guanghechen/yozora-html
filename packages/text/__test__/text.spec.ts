import { TextType } from '@yozora/ast'
import renderText from '../src'

describe('snapshot', function () {
  it('basic', function () {
    expect(
      renderText({
        type: TextType,
        value: 'literal content.',
      }),
    ).toMatchSnapshot()
  })

  it('empty value', function () {
    expect(
      renderText({
        type: TextType,
        value: '',
      }),
    ).toMatchSnapshot()
  })
})
