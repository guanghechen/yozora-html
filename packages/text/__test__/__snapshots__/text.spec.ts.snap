// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot basic 1`] = `"<span class="yozora-text">literal content.</span>"`;

exports[`snapshot empty value 1`] = `"<span class="yozora-text"></span>"`;
